### ISSUE TITLE
Unable to build project depending on Zappa via pip-compile

### ISSUE THREAD

**Issue by jait** [zappa/Zappa#1102]
## Context

Builds of my private project using Zappa 0.54.1 started failing on Fri, Jan 7. No changes were made to the project dependencies.

## Expected Behavior

Build of a project depending on zappa 0.54.1 succeeds as before. 

## Actual Behavior

Build started failing on Fri, Jan 7 with error:

```
#8 22.03 ERROR: Cannot install -r requirements.txt (line 220), -r requirements.txt (line 539) and placebo==0.10.0 because these package versions have conflicting dependencies.
#8 22.03
#8 22.03 The conflict is caused by:
#8 22.03     The user requested placebo==0.10.0
#8 22.03     kappa 0.6.0 depends on placebo>=0.8.1
#8 22.03     zappa 0.54.1 depends on placebo<0.10
```

## Possible Fix

Loosen placebo version requirement.

## Steps to Reproduce

1. Have a requirements.in with zappa==0.54.1
2. pip-compile --generate-hashes --allow-unsafe requirements.in
3. pip install -r requirements.txt

pip-compile generates this in requirements.txt:

```
placebo==0.10.0 \
    --hash=sha256:390db04f3f3486790b583dc18cb0bc116f195f196d4e58195adb4bb543ebe0bd
    # via kappa
```

but since Zappa now requires placebo < 0.10.0, the build fails. 

Why was the package or dependencies of the existing 0.54.1 release changed?

## Your Environment

* Zappa version used: 0.54.1
* Operating System and Python version: Python 3.7

**Comment by jait** [comment]
https://pypi.org/project/zappa/#files shows that the wheel zappa-0.54.1-py3-none-any.whl with sha256 hash 85d6103b5ae936a181e36011dd48ce9c4f7c0603357684e56149faf27fa92cd9 was (re-)uploaded on Jan 7. This seems to cause the problem. 

Two questions:
1) Why was the wheel for an old release (re-)uploaded?

2) If the placebo requirement change (https://github.com/zappa/Zappa/commit/d0d03d56256043707e8fd3f0ba9bdbe5132554c7) was done _after_ 0.54.1 was released, why is it included in the uploaded wheel? Is the wheel in fact built from a later version and not the 0.54.1 tag?

**Comment by cometurrata** [comment]
Hello :wave: 
I had the same issue, freezing `placebo"==0.9.0"` worked for me

**Comment by cometurrata** [comment]
Freezing:

```
zappa = "==0.54.1"
```

In our `pipfile` worked too

**Comment by starlabs007** [comment]
I've encountered this issue as well, pinning `placebo==0.9.0` in my requirements.txt file as per @cometurrata suggestion worked. Thanks.

**Comment by jait** [comment]
Yes, I also worked around this by pinning `placebo<0.10.0` but the more alarming thing is that the zappa-0.54.1 Python 3 wheel on pypi is _not_ in fact version 0.54.1 but seems to contain (unreleased) code from master branch from just few days ago.

Just download it and compare to to different revisions in the source repo: https://files.pythonhosted.org/packages/90/96/15430d603457dbb8a58e35cbc551ce729d758358cbac31466e0886b62c9c/zappa-0.54.1-py3-none-any.whl

 @wrboyce : any comment?

**Comment by javulticat** [comment]
Hi all, you are correct that [`zappa-0.54.1-py3-none-any.whl`](https://files.pythonhosted.org/packages/90/96/15430d603457dbb8a58e35cbc551ce729d758358cbac31466e0886b62c9c/zappa-0.54.1-py3-none-any.whl) on `pypi` is _**not**_ the intended `0.54.1` release and accidentally contains later commits found in `master` (through Jan 7, 2022).  This is an error we are actively working to correct (see: #1103).  In the meantime, [`zappa-0.54.1-py2.py3-none-any.whl`](https://files.pythonhosted.org/packages/d3/92/1e374d7a7b265fcd03440983ac7a907d61192c7be8b92fd4bb09961d93bd/zappa-0.54.1-py2.py3-none-any.whl) _**is**_ the original `0.54.1` release (from Nov 11, 2021), and can be used if `0.54.1` is explicitly needed.  You can also build it yourself from the source on `pypi`, which is also the expected original `0.54.1` release (from Nov 11, 2021): [`zappa-0.54.1.tar.gz`](https://files.pythonhosted.org/packages/a7/22/89f97b38350ee2d7bca2754324c5e4fc88eb5696351026e41e03afc85ada/zappa-0.54.1.tar.gz)

As for what to expect as a future state, we intend to:

1. Merge #1103 to prevent this error from happening again.
2. Release and publish `0.55.0` to `pypi`, which will contain a superset of all of the changes currently found in `zappa-0.54.1-py3-none-any.whl` (the accidental build published on Jan 7, 2022).  The wheel can be expected to be named `zappa-0.55.0-py3-none-any.whl`.
3. Under the `0.54.1` release in `pypi`, delete what is currently called `zappa-0.54.1-py3-none-any.whl` (the accidental build published on Jan 7, 2022) and rename `zappa-0.54.1-py2.py3-none-any.whl` (the original build published on Nov 11, 2021) to `zappa-0.54.1-py3-none-any.whl` (the `-py2.py3-` part of the wheel name is a mistake too - we don't currently support Python 2).  This will bring us to our intended state in `pypi`, where each release has one and only one `py3` wheel, built from a commit marked with an explicit [version tag](https://github.com/zappa/Zappa/tags).

To reiterate, if anyone is relying on features currently found in the accidentally published `zappa-0.54.1-py3-none-any.whl` from Jan 7, 2022, all of those features will be available in the forthcoming `0.55.0` release, which will be published _**before**_ that wheel is removed from `0.54.1` in `pypi`.

Sorry for any confusion or inconvenience this may have caused.  Thank you for bearing with the growing pains of a new group of unpaid maintainers taking over an old, heavily-trafficked, and, until recently, relatively unmaintained repo 😄
